detections/ec2_excessive_runinstances_mltk.yml

```diff
@@ -0,0 +1,97 @@
+asset_type: AWS Instance
+confidence: medium
+creation_date: '2019-11-14'
+data_metadata:
+  data_source:
+    - AWS CloudTrail logs
+  data_sourcetypes:
+    - aws:cloudtrail
+  providing_technologies:
+    - AWS
+description: This search looks for CloudTrail events where a user successfully launches
+  an abnormally high number of instances.
+detect:
```

Comment:
this manifest files needs a "baseline" key. Similar to 

https://github.com/splunk/security-content/blob/73800355367d8b99ff69a9596b908f3bdda76e52/detections/cloud_compute_activity_in_new_region.yml#L2

Reply:
Done in my last commit.